README.md

```diff
@@ -20,6 +20,8 @@ This action will create a GitHub release and optionally upload an artifact to it
 - **omitNameDuringUpdate**: Indicates if the release name should be omitted during updates. The name will still be applied for newly created releases. This will preserve the existing name during updates.
 - **owner**: Optionally specify the owner of the repo where the release should be generated. Defaults to current repo's owner.
 - **prerelease**: Optionally marks this release as prerelease. Set to true to enable.
+- **omitPrereleaseDuringUpdate**: Indicates if the prerelease flag should be omitted during updates. The prerelease flag will still be applied for newly created releases.
```

Comment:
Could we move this up with the rest of the `omit**` inputs? This list is roughly alphabetical.